dev/fbAutoEdits.user.js

```diff
@@ -323,18 +327,25 @@ $(document.body).on('click', '#idDropList_probabilityxofxoccurrencer04_oDropList
     setPriority($(".droplist-text:eq(7)").val(), $(this).text());
 });
 
+$(document.body).ready(function(){
+    console.log("document body ready " + window.location.href);
```

Comment:
Real minor complaint: I want to leave all console.log() stuff in the code for debugging if needed, but commented out by default so I'm not filling the console unnecessarily. I think in the future we'll need to figure out some sort of debug option, but for now just leave that as a comment.

Reply:
Works for me